### DISCUSSION TITLE
💬 Codemagic Developer Community on Discord!

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2223] [Announcements]
We really miss the chit-chat we used to have with the community, so we’ve decided to set up a **Codemagic Community Discord server**! There’s no message limit or chat history that disappears after 90 days, so we think it will make a great place to hang out and connect with the Codemagic Team and other Codemagic users all over the world. In addition to general chat and Community Support, you’ll also be able to see what events we’ll be attending and sponsoring in the coming months. 

## What is Discord?
[Discord](https://discord.com/) is a popular community platform where members can interact and engage with one another via text, video calls, and voice chats.

## How do I access Discord?
Joining Discord is simple. Here’s the process in five steps:

1. [Download the application](https://support.discord.com/hc/en-us/articles/360033931551-Getting-Started) or use a [web browser](https://discord.com/login). 
Note for Mac users, Discord asks for permission to receive keystrokes — which improves accessibility by enabling systemwide push-to-talk (PTT) to work. If you disable this permission, Discord will still function just fine, you’ll just need to open the app for your PTT hotkey to function.
2. [Create an account](https://discord.com/register) If you have a personal account, you can create another profile, or professional account, and switch between the two as you’d like.
3. Check your [privacy settings](https://discord.com/safety/360043857751-four-steps-to-a-super-safe-account).
4. Configure your [notifications](https://support.discord.com/hc/en-us/articles/215253258-Notifications-Settings-101).
5. Join the [Codemagic Discord Server](https://discord.gg/pefznye93R).

Having trouble installing the application or creating an account? Visit the [Discord Help Center](https://support.discord.com/hc/en-us).

We look forward to seeing you on Discord soon!

Cheers!
Team Codemagic

**Comment by ulfie22** [comment]
The Discord invitation comes back as invalid. And searching for a Codemagic server returns nothing. Help!

**Reply by himesh-cm** [reply]
Hi @ulfie22, [Here](https://discord.com/invite/pefznye93R) is the valid link, and I have also updated the announcement.
